cmd/soroban-rpc/internal/test/get_transactions_test.go

```diff
@@ -0,0 +1,141 @@
+package test
+
+import (
+	"context"
+	"testing"
+	"time"
+
+	"github.com/creachadair/jrpc2"
+	"github.com/creachadair/jrpc2/jhttp"
+	"github.com/stellar/go/keypair"
+	"github.com/stellar/go/toid"
+	"github.com/stellar/go/txnbuild"
+	"github.com/stretchr/testify/assert"
+
+	"github.com/stellar/soroban-rpc/cmd/soroban-rpc/internal/methods"
+)
+
+const (
+	Cursor int64 = 55834578945
```

Comment:
it's not clear what's the significance of 55834578945 and how it's used in TestGetTransactions(). How does the test guarantee there will be 2 transactions after this specific cursor?

Reply:
@tamirms I took the cursor string value coming from the previous request's result - https://github.com/stellar/soroban-rpc/pull/136/files/2d660841df55a3e5d5b569c07948ea66aad1481d#diff-dd051ad5301911cdd3964f2129a8d3e93066ca9bd9a9490166fdab6cf28d3952R123.

Reply:
I looked at the code you linked to and I don't see where you extract the cursor out from the response

Reply:
@tamirms I had manually recorded the value of the previous result's cursor and added it to the constant variable. However, I realise that is confusing so I have refactored the code to directly convert the string cursor and use it instead - https://github.com/stellar/soroban-rpc/pull/136/files#diff-dd051ad5301911cdd3964f2129a8d3e93066ca9bd9a9490166fdab6cf28d3952R92

Lmk if this works now. Thanks